app/helpers/basespace_helper.rb

```diff
@@ -17,13 +17,13 @@ def revoke_access_token(access_token)
     )
   end
 
-  def verify_access_token_revoked(access_token)
+  def verify_access_token_revoked(access_token, sample_id)
     # Verify that the token was revoked by using it to call an API endpoint.
     # The API endpoint should return a 401.
 
     fetch_from_basespace(BASESPACE_CURRENT_PROJECTS_URL, access_token, {}, true)
 
-    LogUtil.log_err_and_airbrake("BasespaceAccessTokenError Revoke access token check failed")
+    LogUtil.log_err_and_airbrake("BasespaceAccessTokenError: Failed to revoke access token for sample id #{sample_id}")
   rescue
     # The call should fail.
     Rails.logger.info("Revoke access token check succeeded")
```

Comment:
I know it is not part of this PR, but I feel like this rescue block should be refactored.
First, it is really counterintuitive this logic like "if it fails then it is a success, and if it succeeds then it is an error"
Second, the rescue block should to catching a very specific error, otherwise it will mask other situations. For example: what if it returned a 500? or if access token is null? All these situations would be considered as success, when in fact it should be an error.

Reply:
Ideally, if status_code is important in this case, `fetch_from_basespace` should be able to return that status and not fail with an error in this case. It could be a parameter that could modify its behavior, like `return_failed_status_codes=True` (we can find a better name) and then you can parse this response code.

Reply:
This is a great point. Thanks for pointing this out.